packages/libs/components/src/stories/plotControls/EzTimeFilter.stories.tsx

```diff
@@ -0,0 +1,360 @@
+        },
+        {
+          N: 1893,
+        },
+        {
+          N: 1913,
+        },
+        {
+          N: 1916,
+        },
+        {
+          N: 1882,
+        },
+        {
+          N: 1967,
+        },
+        {
+          N: 1699,
+        },
+        {
+          N: 1697,
+        },
+        {
+          N: 1263,
+        },
+        {
+          N: 1774,
+        },
+        {
+          N: 1620,
+        },
+        {
+          N: 1450,
+        },
+        {
+          N: 1893,
+        },
+        {
+          N: 1628,
+        },
+        {
+          N: 1552,
+        },
+        {
+          N: 1722,
+        },
+        {
+          N: 1698,
+        },
+        {
+          N: 1604,
+        },
+        {
+          N: 1544,
+        },
+        {
+          N: 1368,
+        },
+        {
+          N: 1302,
+        },
+        {
+          N: 1250,
+        },
+        {
+          N: 1254,
+        },
+        {
+          N: 751,
+        },
+        {
+          N: 527,
+        },
+        {
+          N: 267,
+        },
+        {
+          N: 51,
+        },
+      ],
+      name: 'Data',
+      mode: 'lines+markers',
+      opacity: 0.7,
+      marker: {
+        color: 'rgb(136,34,85)',
+        symbol: 'circle',
+      },
+    },
+  ],
+};
+
+export const TimeFilter: Story<LinePlotProps> = (args: any) => {
+  // converting lineplot data to visx format
+  const timeFilterData: EZTimeFilterDataProp[] = LineplotData.series[0].x.map(
+    (value, index) => {
+      // return { x: value, y: LineplotData.series[0].y[index] };
+      return { x: value, y: LineplotData.series[0].y[index] >= 9 ? 1 : 0 };
+    }
+  );
+
+  // set initial selectedRange
+  const [selectedRange, setSelectedRange] = useState({
+    start: timeFilterData[0].x,
+    end: timeFilterData[timeFilterData.length - 1].x,
+  });
+
+  // set forwardRef to call handleResetClick function from EzTimeFilter component
+  const childRef = useRef<{ handleResetClick: () => void }>(null);
+
+  // set constant values
+  const defaultSymbolSize = 0.8;
+  const defaultColor = 'lightgray';
+
+  return (
+    <div
+      style={{
+        width: 750,
+        height: 170,
+        border: '1px solid #dedede',
+        boxShadow: '1px 1px 4px #00000066',
+        marginTop: '10em',
+        marginLeft: '10em',
+      }}
+    >
+      <div
+        style={{
+          display: 'grid',
+          gridTemplateColumns: '1fr repeat(1, auto) 1fr',
+          gridColumnGap: '5px',
+          justifyContent: 'center',
+          paddingTop: '1em',
+        }}
+      >
+        {/* display start to end value */}
+        <div style={{ gridColumnStart: 2 }}>
+          {selectedRange?.start} ~ {selectedRange?.end}
+        </div>
+        {/* button to reset selectedRange */}
+        <div
+          style={{
+            marginLeft: 'auto',
+            marginTop: '-0.1em',
+            paddingRight: '1.5em',
+          }}
+        >
+          <a
```

Comment:
So, my (initial) intention to place the reset at parent level was for positioning. But I think that it may be better to place the reset and selected range label into the ez time filter component, and then adjust InputVariable component, which should be at parent component, via CSS positioning. 

Yes, I also noticed that <a> tag causes it at storybook. Will handle it, either change to button or something else.